en/lessons/creating-mobile-augmented-reality-experiences-in-unity.md

```diff
@@ -0,0 +1,411 @@
+
+HP Reveal is a fantastic AR creation platform that can be learned fairly quickly. I would highly recommend it for classroom usage and for smaller projects with simple multimedia overlays such as images or short videos. However, online creation platforms like HP Reveal have their limitations. When you create an overlay in HP Reveal, users can only access it through the HP Reveal application. This leaves the AR creator with no control over the design of the application interface. Moreover, HP Reveal requires that users have internet acess when using the application, which makes it difficult to create site-specific AR applications in places where there is little to no internet connectivity. With Vuforia, however, you can create stand-alone AR applications that can be downloaded and stored in the mobile device's memory and thus do not require that users to have internet access. Consult the table below to help you decide which platform is best suited for your AR development needs.
+
+|            | HP Reveal        | Unity/Vuforia    |     
+|------------| ------------- |-------------|
+| **Type**           | online platform    | computer program |
+| **Use**           | small projects with simple overlays     | large projects with complex overlays, site-specific applications |
+| **Pros**           | users can instantly access overlays, no coding required    | create standalone applications, more control over interface design, overlays load quickly |
+| **Cons**           | file size limits for overlays, requires internet connection     |steeper learning curve, must upload completed application files to Google Play or Apple Store for others to access|
+| **Access**           | [HP Reveal Studio](https://www.hpreveal.com/) | [Unity](https://unity3d.com/get-unity/download/archive) [Vuforia](https://developer.vuforia.com) |
+| **Minumum Hardware Needed**           | iOS or Android Mobile Device    | Laptop/Desktop and iOS or Android Mobile Device |
+
+
+### Installing Unity and Vuforia
+
+Since the release of Unity 2017.2, the Vuforia SDK is integrated into the Unity Editor. The [Vuforia SDK](https://developer.vuforia.com/) is a suite of assets and code snippets that work alonside Unity to make it easier to quickly develop and build AR content for a variety of platforms. If you are unable to download Unity 2017.2 or later, [consult this archived lesson for earlier versions of Unity](link to previous lesson). To download Unity and Vuforia, go to the [Unity website](https://unity3d.com/get-unity/download/archive) and download Unity 2017.2. In the Components dialog box, select Vuforia Augmented Reality Support in addition to either Android Build Support and/or iOS Build Support, depending on your target mobile device platform. Once the download completes, start Unity and follow the setup prompts. If Unity asks if you are creating a personal or professional account, choose personal account.
```

Comment:
the `(link to previous lesson)` needs a URL

Reply:
Yep just noticed that. Will fix in a sec.